lib/Common/CommonPal.h

```diff
@@ -274,6 +274,10 @@ inline T InterlockedDecrement(
     return __sync_sub_and_fetch(Addend, T(1));
 }
 
+inline __int64 _abs64(__int64 n)
```

Comment:
My mistake, I thought you were talking about STL.

Yes, _abs64 is on MS system, but not on non _WIN32. I added it to non _WIN32 only.

You are right I could use llabs. I couldn't find existing usage of it in the project. Since `long long` doesn't guarantee to be same as `__int64`, a version for __int64 might be ok?